setup.py

```diff
@@ -5,48 +5,76 @@
 
 
 PACKAGE_NAME = "netcdf-scm"
+DESCRIPTION = "Processing netCDF files for use with simple climate models"
+KEYWORDS = [
+    "netcdf",
+    "netCDF",
+    "python",
+    "climate",
+    "atmosphere",
+    "simple climate model",
+    "reduced complexity climate model",
+    "data processing",
+]
+
 AUTHOR = "Zebedee Nicholls"
 EMAIL = "zebedee.nicholls@climate-energy-college.org"
 URL = "https://github.com/znicholls/netcdf-scm"
+PROJECT_URLS = {
+    "Bug Reports": "https://github.com/znicholls/netcdf-scm/issues",
+    "Documentation": "https://openscm.readthedocs.io/en/latest",
+    "Source": "https://github.com/znicholls/netcdf-scm",
+}
+LICENSE = "2-Clause BSD License"
+CLASSIFIERS = [
+    "Development Status :: 4 - Beta",
+    "License :: OSI Approved :: BSD License",
+    "Intended Audience :: Developers",
+    "Operating System :: OS Independent",
+    "Programming Language :: Python :: 3.7",
+]
 
-DESCRIPTION = (
-    "Python wrapper for processing netCDF files for use with simple climate models"
-)
-README = "README.rst"
+ENTRY_POINTS = {"console_scripts": ["netcdf-scm-crunch = netcdf_scm.cli:crunch_data"]}
```

Comment:
Why not just netcdf-scm? Are you planning on having multiple CLI tools?

Reply:
yes